backend/apps/residents/services.py

```
@@ -142,8 +142,11 @@ def create_resident(data: dict, residence, request) -> dict:
     else:
         try:
             process_password_reset_request(user.email, request)
-        except Exception:
-            pass
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)